File: JavaScript/Resolvendo desafios em Javascript/xadrez.js
let lines = gets().split("\n");

var l = parseInt(lines[0]);
var c = parseInt(lines[1]);

if (l % 2 === 0)
  if (c % 2 === 0) print(1);
  else print(0);
else if (l % 2 === 1) print(1);
 else print(0);      //complete o código nos espaços em branco

File: Java/Desafios Iniciais - GFT Start #3 Java/Xadrez.java
import java.util.Scanner;

public class Xadrez {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int L = sc.nextInt();
        int C = sc.nextInt();
        if (L % 2 == 0)
            if (C % 2 == 0)
                System.out.println("1");
            else
                System.out.println("0");
        else if (L % 2 == 1)
            System.out.println("1"); // complete o código nos espaços em branco
        else
            System.out.println("0");
        sc.close();
    }
}
